webcompat/form.py

```diff
@@ -118,9 +119,24 @@ def get_labels(browser_name):
 def normalize_url(url):
     '''normalize URL for consistency.'''
     url = url.strip()
-    if not url.startswith(SCHEMES):
+    parsed = urlparse.urlparse(url)
+
+    if url.startswith(BAD_SCHEMES):
+        # if url starts with a bad scheme, parsed.netloc will be empty,
+        # so we use parsed.path instead
+        path = parsed.path.lstrip('/')
+        url = '%s://%s' % (parsed.scheme, path)
```

Comment:
nit: can we use `str.format` style (and elsewhere) here instead of the old `%` `sprintf` style of string interpolation here?

(I know `normalize_url` was already using it, but generally I think `'{0://{1}'.format(parsed.scheme, path)` is easier to read).

See https://docs.python.org/2/library/stdtypes.html#str.format